hooks/useVestingContract.ts

```diff
@@ -107,17 +107,19 @@ export const useVestingPeriod = (address: string) => {
   return { startInTime, endInTime, isLoading };
 };
 
-export const useVestingClaim = (address: string) => {
+export const useVestingClaim = (address = '') => {
   const { contractWeb3, account, chainId } = useVestingEscrowContract(address);
 
   return useCallback(
-    async (value: string) => {
-      if (!contractWeb3 || !account || !chainId) return;
-
-      const callback = () =>
-        contractWeb3['claim(address,uint256)'](account, parseEther(value));
-
-      await transaction('Claim', chainId, callback);
+    async (amount: string, selectedAccount = account) => {
+      if (!contractWeb3 || !selectedAccount || !chainId) return;
+
+      await transaction('Claim', chainId, () =>
```

Comment:
will be good add catch for displaying valid error

Reply:
Will do in another PR https://linear.app/lidofi/issue/UI-470/fix-reject-error-message